src/server.jl

```diff
@@ -70,6 +70,8 @@ function _exeflags_and_env(options)
     if !any(startswith("JULIA_PROJECT="), env) && !any(startswith("--project="), exeflags)
         push!(exeflags, "--project=@.")
     end
+    # if exeflags already contains '--color=no', the 'no' will prevail
+    pushfirst!(exeflags,"--color=yes")
```

Comment:
Related to #69 I assume? Can this be moved to a separate PR, which we'll be able to review and merge separately.

Reply:
oops, I removed the second commit